pkgs/development/tools/ttfb/default.nix

```diff
@@ -0,0 +1,36 @@
+{ fetchCrate, rustPlatform, lib, pkg-config, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ttfb";
+  version = "1.6.0";
+
+  cargoSha256 = "sha256-ayyYrrFDVOYVjVo5TLaRn2mvmywe5BjQ7kRVV2r0iK8=";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-o7kzQ8jtAqDwTUPtjeNqgotxREeWl7jQG+EDrYWJL/Q=";
+  };
+
+  # The bin feature activates all dependencies of the binary. Otherwise,
+  # only the library is build.
+  buildFeatures = ["bin"];
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ];
+
+  meta = with lib; {
+    description = ''
```

Comment:
meta.description should be a string, not a mutliline string because search.nixos.org only handles one pine properly in the layout.